### PR TITLE
Fix Insecure Script / Content Execution and BigInt not working properly 

### PR DESCRIPTION
**Pull request by sebastian93921**
👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [x] Have you read the [CONTRIBUTING.md](/CONTRIBUTING.md)?
- [x] Have you rebased your work on top of the latest `dev`? 
- [x] Does your PR follow the [Conventional Commits Guide](https://www.conventionalcommits.org/)?
- [x] Have you checked your code compiles? (`yarn build`)
- [x] Have you included tests for any non-trivial functionality?
- [x] Have you followed the [Multi-Language Content Management Guide](/CONTENT_MANAGEMENT.md)?
- [x] Have you checked your code passes the unit tests? (`yarn test`)
- [x] Have you checked your code formatting is correct? (`yarn lint:js`)
- [x] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`yarn audit`)
- [x] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](/CONTRIBUTING.md#developer-certificate-of-origin).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by sebastian93921 — COMMENTED** [review]
Fix TypeError: Do not know how to serialize a BigInt

**Review by sebastian93921 — COMMENTED** [review]
Bugfix on BigInt based on the following:
https://github.com/paulmillr/noble-ed25519/issues/23

**Review by crypto-matto — APPROVED** [review]
Much appreciated your work! @sebastian93921

File: electron/main.ts [hunk 1]
```diff
@@ -148,6 +148,15 @@ function createWindow() {
     require('electron').shell.openExternal(finalURL);
   });
 
+
+  win.webContents.on('did-start-navigation',(e, url, isInPlace, isMainFrame) => {
```
Comment (XinyuCRO) [inline electron/main.ts:152]: Seems this event only notifies the navigation, no way to stop it here?

**Review by XinyuCRO — APPROVED** [review]
LGTM

File: electron/main.ts (hunk 1 above)
Comment (sebastian93921) [reply electron/main.ts:152]: I cannot trigger this event normally too, just in case any kind of external URL comes in place

Comment (crypto-matto) [reply electron/main.ts:152]: Purpose-wise this PR achieves the goal. I will merge this PR now.

**Pull request merged**